packages/odyssey-react-mui/src/labs/DataComponents/DataView.tsx

```diff
@@ -0,0 +1,388 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { memo, useEffect, useMemo, useState } from "react";
```

Comment:
I don't know how we all feel about this but, this is my preferred way of doing imports
```js
import { memo, useEffect, useMemo, useState } from "react";
import {
  MRT_Row,
  MRT_RowData,
  MRT_RowSelectionState,
} from "material-react-table";
import { t } from "i18next";

import { Box } from "../../Box";
import { BulkActionMenu } from "./BulkActionsMenu";
import { Callout } from "../../Callout";
import {
  Layout,
  UniversalProps,
  ViewProps,
  TableState,
} from "./componentTypes";
import {
  availableLayouts as allAvailableLayouts,
  densityValues,
} from "./constants";
import { useRowReordering } from "../../DataTable/useRowReordering";
import { DataFilter, DataFilters } from "../DataFilters";
import { EmptyState } from "../../EmptyState";
import { fetchData } from "./fetchData";
import { LayoutSwitcher } from "./LayoutSwitcher";
import { Pagination, usePagination } from "../../Pagination";
import { StackContent } from "./StackContent";
import { TableSettings } from "./TableSettings";
import { TableContent } from "./TableContent";
import { useFilterConversion } from "./useFilterConversion";
```
Where 'their' stuff is all imported at the top. Then an empty line. Then 'our' stuff imported alphabetically.

I could also argue for a method in which we sort our imports alphabetically AND by relative path distance. Closest coming first. Any thoughts? cc// @KevinGhadyani-Okta @benlister-okta

Reply:
I like the format and line for distinction from the non-odyssey files. I don't have a strong preference for ordering our odyssey or internal components alphabetically by `{ item }` or path as long as we're consistent but I would REALLY love if there was a way to auto-format this with a linter rule